src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/dispenser.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "modern_industrialization:assembler",
+  "eu": 8,
+  "duration": 200,
+  "item_inputs": [
+    {
+      "item": "minecraft:cobblestone",
+      "amount": 7
+    },
+    {
+      "item": "minecraft:string",
```

Comment:
Conflicts with the dropper

Reply:
Hmm... Gonna make the recipes into
Dropper:
6 Cobble
1 Redstone
1 Bronze Ring

Dispenser
6 Cobble
1 Redstone
1 String
1 Iron Ring
Basically give them nozzles.

Reply:
What rings to use is not final. Considering using Invar for one tho.

Reply:
Done.